app.py

```diff
@@ -116,6 +120,16 @@ def main():
                               + "not be reached or read.",
                               True, True)
 
+    # Check if the path to the post-processing external script exists and if the folder is accessible
+    if (post_process_script_command != "Bagger"):
+        post_process_script_path_exists = os.path.exists(post_process_script_command)
+        post_process_script_folder_access = os.access(post_process_script_command, os.W_OK)
+        if (post_process_script_path_exists is False or post_process_script_folder_access is False):
+            log.write_log_in_file('error',
+                                  "The post process script location specified in the config file could"
+                                  + "not be reached or read.",
+                                  True, True)
```

Comment:
Setting the final argument to `write_log_in_file` to True will stop the script from executing if there is an error. To keep the behavior intended by 4.5.1 of the specification, execution should continue by setting it to False. 

The spec says that if there is an error with the script, an error should be shown and the current code path aborted, but the rest of ReBACH should continue. Since the current code path is the main code path, this may be interpreted to abort all of ReBACH but this isn't what we want in this case.

Reply:
Updated final argument to write_log_in_file to False